### ISSUE TITLE
In plugin package installations, irods-owned paths may become root-owned

### ISSUE THREAD

**Issue by d-w-moore** [irods/irods#5679]
### iRODS Version, OS and Version
iRODS 4.2.9 + Python REP 4.2.9.0 (just pre-release) and greater on  Centos 7
*BUT* this may apply to many other plugins, for all we know....

### What did you try to do?

Install an iRODS plugin ( in this case the Python rule engine plugin )  RPM that writes files to `/etc/irods` or `/var/lib/irods` ...

### Expected behavior

These directories should remain owned by irods service account user.

### Observed behavior (including steps to reproduce, if applicable)

The directories become root-owned, preventing the `irods` user from being able to create new files within them. (As hinted, this was first discovered while running Python REP tests on Centos 7).

### Explanation:
Because of a snag in CMake RPM defaults, whenever files are installed from a package, the paths "above" them are also marked as install targets.

**Comment by trel** [comment]
so this belongs in the PREP repo, yes?    no changes needed in the core?

**Comment by d-w-moore** [comment]
yes. I thought it belonged in irods because other plugins could be affected ....

**Comment by d-w-moore** [comment]
but yes,correct. no changes in the iRODS core code.
